src/HeadCount/HeadCount.js

```diff
@@ -32,8 +34,12 @@ export default class HeadCount extends Component {
   }
 
   handleClick(location) {
-    console.log(location+ 'this was clicked');
-
+    if(this.state.compare.length<2){
+      this.state.compare.push(this.state.data.findByName(location))
+    }
+    this.state.compare.shift()
```

Comment:
shift first item out of array if you are trying to add and the array already has 2 in it.